src/index.html

```diff
@@ -11,7 +11,184 @@
+            maxlength="3"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            required
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox"
+            name="terms"
+            id="terms-and-confitions"
+            required
+          >
+          <label for="terms-and-confitions"></label>
+        </label>
+      </fieldset>
+      
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            minlength="15"
+            maxlength="80"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="32"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="y"
+              required
+            >
+            Yes
+          </label>
+    
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="n"
+              required
+            >
+            No
+          </label>
+        </label>
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            required
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            required
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="car"
+            multiple
+            required
+          >
+            <option>BMW</option>
+            <option>Audi</option>
+            <option>Lada</option>
+            <option disabled></option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            value="0"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            maxlength="1000"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            maxlength="1000"
          ></textarea>
```

It should fix spaces in textarea field

Reply:
when i do like this (npm run lint) gives me an error:
![image](https://user-images.githubusercontent.com/121498080/219685446-5bd40c0a-8984-43f0-8ab3-be8d334e6638.png)
when i revert back i have no errors:
![image](https://user-images.githubusercontent.com/121498080/219686069-dd515784-b1b7-4b67-88da-c53423d7dca8.png)

Reply:
![image](https://user-images.githubusercontent.com/121498080/219713871-120e0ab7-8cd7-4727-96f7-0734c58f5f3c.png)

Reply:
Thanks for your help Nazar! Issue resolved!